raiden-cli/raspberrypi.md

```diff
@@ -0,0 +1,92 @@
+# Running raiden-cli on the raspberry pi
+
+This tutorial assumes raspberry pi to be in the same LAN as your PC with SSH enabled
+These steps have to be run on the raspberry pi 4. The steps to copy files from your `PC` to the `pi` have been explicitly mentioned and have to be run on your `PC`.
+
+`sudo apt update`
+
+`sudo apt full-upgrade`
+
+`sudo apt install git`
+
+Perform a [git install](https://github.com/nvm-sh/nvm#git-install) of the NVM
+
+`git clone https://github.com/nvm-sh/nvm.git .nvm`
+
+`cd ~/.nvm`
+
+`git checkout v0.38.0`
+
+`. ./nvm.sh`
+
+`nvm install --lts`
+
+`npm install --global yarn`
+
+Check the arm system
+
+```
+uname -m
+armv7l
+```
+
+`wget https://gethstore.blob.core.windows.net/builds/geth-linux-arm7-1.10.4-aa637fd3.tar.gz`
+
+`tar -xvzf geth-linux-arm7-1.10.4-aa637fd3.tar.gz`
+
+`sudo cp geth-linux-arm7-1.10.4-aa637fd3/geth /usr/local/bin/`
+
+`which geth`
+
+`geth version`
+
+`geth account new`
+
+Refer `raiden-cli` [README](https://github.com/raiden-network/light-client/tree/master/raiden-cli) to generate the bundle folder on your x64 PC.
```

Comment:
Refer *to*

Reply:
actually there was more to it. So corrected it